app/build.gradle.kts

```diff
@@ -43,7 +43,23 @@ android {
         generateLocaleConfig = true
     }
 
+    signingConfigs {
+        getByName("debug") {
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+            storeFile = file("../keystores/debug.keystore")
+            storePassword = "android"
+        }
+    }
+
     buildTypes {
+
```

Comment:
To match our other gradle files we can omit the extra newline here.

Reply:
Cool 👍 

b8b85b7657b03bfd049131e81b6a87291098ecc8